resources/schema.json

```diff
@@ -135,7 +135,10 @@
         },
         "mutators": {
             "type": "object",
-            "additionalProperties": false,
+            "additionalProperties": {
```

Comment:
as Theo already mentioned [here](https://github.com/infection/infection/pull/947#discussion_r363245788), this change can lead to incorrect (not so strict) usage of configuration files. Probably we need to move it inside some key, like `mutators.custom`? I don't know

Reply:
resolved here: https://github.com/infection/infection/pull/1686/commits/2f6ec735b7633ed40472ecfa03ccc709618a54ac